### ISSUE TITLE
missing cookies or session

### ISSUE THREAD

**Issue by saggit** [fengsp/pencil#6]
cookies or session is necessary to build an web app with authenticate.

**Comment by TheWaWaR** [comment]
I think the author will implement cookies use [hyper::header::Cookie](http://hyper.rs/hyper/hyper/header/struct.Cookie.html)

**Comment by TheWaWaR** [comment]
@saggit I believe [this](https://github.com/fengsp/pencil/commit/f52666fe4353177fedfd9b03113adf21e3bfd63b) is what you want.

**Comment by fengsp** [comment]
@saggit Yes, I don't have time to implement the sessions yet.  We should add it in the future releases.
